src/components/http/HttpService.brs

```diff
@@ -51,6 +51,12 @@ function HttpService(port as Object, httpInterceptors = [] as Object) as Object
     return m._waitForResponse(request, cachedResponse)
   end function
 
+  ' Clears the cache in scope of this service
+  ' @returns {Boolean} - true if scoped cache has been cleared
+  prototype.clearCache = function () as Boolean
```

Comment:
I'm wondering if this is needed - not used by any other method. From the outside of this entity, HttpCache could be used directly

Reply:
reverted in 267cc632eda206f3aeb3f6f80ba17748a122893d as per our private discussion 🙂